DataFormats/BTauReco/interface/TemplatedSecondaryVertexTagInfo.h

```diff
@@ -58,7 +58,7 @@ class TemplatedSecondaryVertexTagInfo : public BaseTagInfo {
 
         struct VertexData {
                 VTX                             vertex;
-                Measurement1D                   dist2d, dist3d;
+                Measurement1D                   dist1d,dist2d, dist3d;
                 GlobalVector                    direction;
 		
 		// Used by ROOT storage
```

Comment:
In classes_def.xml file right?
Also there may be some older version which should not be replaced. I mean I will keep the other version as it is but will add version 12 for Measurement1D. Please do correct me if I am wrong.